lib/persistentqueue/persistentqueue.go

```diff
@@ -202,7 +202,13 @@ func tryOpeningQueue(path, name string, chunkFileSize, maxBlockSize, maxPendingB
 		fs.MustWriteAtomic(filepath, nil, false)
 	}
 	if mi.Name != q.name {
-		return nil, fmt.Errorf("unexpected queue name; got %q; want %q", mi.Name, q.name)
+		// For compatibility, update metadata name to new format.
+		// https://github.com/VictoriaMetrics/VictoriaMetrics/issues/8477
+		logger.Infof("a compatible metadata file was found: %s. Update the metadata name from %q to %q.", metainfoPath, mi.Name, q.name)
```

Comment:
I think, it's not safe to log queue name. Since it may have sensitive information now. Previously, it could be hidden with `showURL=false`. 

 Lets just remove this path
```
if mi.Name != q.name {
   ...
 }
```

 metaInfo will be eventually update to the new name. I also think we should deprecate q.Name, since it's not used anymore.

Reply:
OK. After this commit, queue verification will rely only on the seq + hash value of the folder name.